dbcon/mysql/ha_calpont_impl.cpp

```diff
@@ -5275,4 +4878,475 @@ int ha_calpont_impl_group_by_end(ha_calpont_group_by_handler* group_hand, TABLE*
     return rc;
 }
 
+
+/*@brief  Initiate the query for derived_handler           */
+/***********************************************************
+ * DESCRIPTION:
+ * Execute the query and saves derived table query.
+ * There is an extra handler argument so I ended up with a
+ * new init function. The code is a copy of
+ * ha_calpont_impl_rnd_init() mostly. 
+ * PARAMETERS:
+ * mcs_handler_info* pnt to an envelope struct 
+ * TABLE* table - dest table to put the results into
+ * RETURN:
+ *    rc as int
+ ***********************************************************/
+int ha_cs_impl_pushdown_init(mcs_handler_info* handler_info, TABLE* table)
+{
+#ifdef DEBUG_SETENV
+    string home(getenv("HOME"));
+
+    if (!getenv("CALPONT_HOME"))
+    {
+        string calpontHome(home + "/Calpont/etc/");
+        setenv("CALPONT_HOME", calpontHome.c_str(), 1);
+    }
+
+    if (!getenv("CALPONT_CONFIG_FILE"))
+    {
+        string calpontConfigFile(home + "/Calpont/etc/Columnstore.xml");
```

Comment:
Whatever this is, please tell me it is wrong. We shouldn't have 'Calpont' in new paths and it isn't a valid old path.

Reply:
This is an outdated way of changing module role for debugging purposes in CSC catalog connections. We don't use this mechanism so I will remove them.